### PR TITLE
Refactored initial code to be more Angular-like

### PR DESCRIPTION
**Pull request by Pikkuninja**
Binded mapdivs size to $rootScope.window_dimensions in html, moved map init to controller scope

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -18,15 +18,15 @@
...
+	<div layout='column' ng-controller='MainController' ng-if='$root.initialized' >
 		<div flex layout='row' class='md-padding' layout-fill ng-if='!$root.fullscreen'>
 			<div flex layout='column' layout-align='center center'>
 				<span class='md-display-1' style='text-align: center;'>City Information</span>
 			</div>
 		</div>
 
 		<div flex layout='row' layout-fill ng-show='$root.fullscreen'>
-			<div id='gmap'></div>
+			<div id='gmap' style="width: {{window_dimensions.fullscreen_width}}px; height: {{window_dimensions.fullscreen_height - 64}}px;"></div>
```
Comment (antti-TIEA4) [inline app/index.html]: Maybe these should go to: https://github.com/APT-project/city_info/blob/master/app/css/style.css ?

Comment (Pikkuninja) [inline app/index.html]: That would mean that all the parents need to be changed to have width&height of 100% too, which probably isn't that bad. This just seemed like something that an Angular dev would do :D. In the devkits index there's couple of places where they do similar things, though they used ng-style attribute.

Comment (Pikkuninja) [reply app/index.html]: Did some testing. It is possible to fill the screen with map by making all containers have w&h of 100%. But this makes centering the map to a certain point during the tile -> fullscreen transition impossible, as the center of div after transitioning is over is unknown. Or maybe it's not impossible but it's way more annoying that way.

**Comment by Pikkuninja** [comment]
Made the current way atleast a bit nicer with the ng-style and storing the actual app dimensions for others to use also.
Switched my tabs to spaces as it seems that the example used spaces for indentation.

File: app/js/app.js [hunk 2]
```diff
@@ -7,129 +7,80 @@ pw.config(function($mdThemingProvider) {
...
-			$rootScope.$apply(function(){
-				$rootScope.fullscreen = true;
-        setMapSize($rootScope.window_dimensions.fullscreen_width, $rootScope.window_dimensions.fullscreen_height);
+  llb_app.request('window_dimensions')
+  llb_app.addListener('window_dimensions', function(data){
+    $rootScope.$apply(function(){
+      $rootScope.window_dimensions = data
+      $rootScope.fullscreen_app_dimensions = {
+        "width": data.fullscreen_width + "px",
+        "height": data.fullscreen_height - 64 + "px"
```
Comment (antti-TIEA4) [inline app/js/app.js]: Wondering if Angular provides a way to get the toolbar height (`64`) somehow?

Comment (Pikkuninja) [inline app/js/app.js:34]: In my opinion the data that the platform gives us should already have the toolbar height removed from the fullscreen_height, as we can't use that space anyhow anyway.

**Pull request merged**